changelogs/fragments/vault_cli_fix.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - ansible-vault will now correclty handle `--prompt`, previouslly it would issue an error about stdin if no 2nd argument was passed
```

Comment:
```suggestion
  - ansible-vault will now correctly handle `--prompt`, previously it would issue an error about stdin if no 2nd argument was passed
```